internal/controller/sparkapplication/submission.go

```diff
@@ -1005,6 +1001,10 @@ func dynamicAllocationOption(app *v1beta2.SparkApplication) ([]string, error) {
 		args = append(args, "--conf",
 			fmt.Sprintf("%s=%d", common.SparkDynamicAllocationMaxExecutors, *dynamicAllocation.MaxExecutors))
 	}
+	if dynamicAllocation.ShuffleTrackingEnabled != nil {
+		args = append(args, "--conf",
+			fmt.Sprintf("%s=%t", common.SparkDynamicAllocationShuffleTrackingEnabled, *dynamicAllocation.ShuffleTrackingEnabled))
+	}
```

Comment:
```suggestion
	if dynamicAllocation.ShuffleTrackingEnabled != nil {
		args = append(args, "--conf",
			fmt.Sprintf("%s=%t", common.SparkDynamicAllocationShuffleTrackingEnabled, *dynamicAllocation.ShuffleTrackingEnabled))
	} else {
	  	args = append(args, "--conf",
		    fmt.Sprintf("%s=true", common.SparkDynamicAllocationShuffleTrackingEnabled))
	}
```

Since we are going to deprecate the webhook in the future, it would be better if it can work without the webhook.